common/match/pool.ts

```diff
@@ -533,9 +537,11 @@ export async function predictPupilMatchTime(pool: MatchPool, averageMatchesPerMo
     // as they were not yet asked for an interest confirmation
     // From those we lose about a third of pupils as they do not confirm their interest
     // This needs to be factored in, as it reduces the actual waiting time
-    if (pool.toggles.includes('skip-interest-confirmation')) {
+    if (pool.toggles.includes('confirmation-pending')) {
         backlog +=
-            ((await getPupilCount(pool, ['confirmation-pending'])) + (await getPupilCount(pool, ['confirmation-unknown']))) *
+            ((await getPupilCount(pool, ['confirmation-pending'])) +
+                (await getPupilCount(pool, ['confirmation-pending'])) +
```

Comment:
Shouldn't this be `confirmation-success`?

Reply:
No, the pupils with confirmation-success are already in the backlog, but yes, this line is wrong and I accidentally changed it, good catch!. Adapted it to also calculate pending pupil screenings